### ISSUE TITLE
Aug 18 - Introducing the Azure ML SDK

### ISSUE THREAD

**Issue by drguthals** [microsoft/ReactorPlanning#28]
# Event Information
| Category | Details |
|-----------|---------|
| Reactor Topic | Data Science and Machine Learning |
| Monthly Topic (if applicable) | Powerful APIs and Power Platform |
| Weekly Topic (if applicable) | Azure ML Python SDK |
| Stream Topic | Sarah's going to talk through the AML SDK |
| Stream Date | Aug 18, 2020 | 
| Stream Time | 4pm PT | 
| Stream Duration | 60 min | 
| Title | Introducing the Azure ML SDK |
| Description | The Azure Machine Learning (ML) SDK creates a way for developers to remain in the familiar environment of VS Code while using the power of Azure to run ML experiments and even deploy ML models. In this session, we’ll explore the lifecycle of datasets used in ML experiments, train models using local or cloud-based resources, and use automated ML.  |
| Speaker | Sarah Guthals, PhD and/or TBD |
| Moderator | <Name or GitHub alias> |
| Event ID | 7852 |

Additional Resources:  
<Additional resources that can be linked from social media for promotion or added to the description of VOD>

# Event Checklist

## 10 Weeks Prior
- [X] Reactor Content Team (RCT) or Speaker opens a new issue for an upcoming stream filling in the Event Information table. Both RCT and Speaker are marked as assignees.
- [X] RCT and Speaker discuss details of the event in the comments of this issue. RCT adds a _confirmed-event_ label and _global-twitch-event_ tag to the issue and adds event to the __Global Live Streams__ project. Issue will appear in the __Upcoming Streams__ column
- [X] Speaker adds a _ready-for-marketing-edit_ label so the RCT can review title and description

## 8 Weeks Prior
- [x] RCT does a review of title and description and removes the _ready-for-marketing-edit_ label 
- [x] RCT moves the issue to the __Ready for SharePoint__ column
- [x] RCT informs the Reactor Marketing Team of the new event information
- [ ] RCT provides speaker with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT moves the issue to the __MONTH Streams Prep__ column (according to the month it will be streamed in)

## 2 Weeks Prior *(Done in /microsoft/Reactors)*
- [ ] Speaker opens a PR with the new content for the stream on the [customer-facing Reactor repository](https://github.com/microsoft/Reactors) and adds the _content-development_ and _global-twitch-event_ tag to the PR. 
- [ ] Speaker adds to the same PR a modification to the [Streaming README](https://github.com/microsoft/Reactors/tree/main/Online) to include their stream information.
- [ ] Speaker adds the RCT assigned to the issue as the reviewer when it is ready for review/merge
- [ ] RCT reviews, approves, and merges the changes

## 1 Week Prior
- [ ] Speaker and Moderator test setup for streaming (instructions coming soon)
- [ ] RCT checks in with the speaker and moves the issue to the __Ready to Stream__ column

## During and After Stream
- [ ] Speaker and Moderator run the stream
- [x] RCT submits recording to be uploaded to the Reactor YouTube and to be made available on LearnTV
- [x] RCT moves issue to the __Stream Complete__ column

**Comment by drguthals** [comment]
For reference: https://docs.microsoft.com/en-us/python/api/overview/azure/ml/?view=azure-ml-py

**Comment by KimberlyBolton1** [comment]
Reached out to Sarah for more content to edit.

**Comment by KimberlyBolton1** [comment]
@sguthals
   I was reading through the draft abstract for AML SDK. Before I provide an edit, is there any more content you’d like to include? I was thinking you could add some details on what you plan to discuss specifically so attendees have a clear expectation. I have content I found that I can add but didn’t want to go too far in a direction that you didn’t intend. But as a starting point, will you cover any of the following?
•	Explore, prepare and manage the lifecycle of your datasets used in machine learning experiments.
•	Manage cloud resources for monitoring, logging, and organizing your machine learning experiments.
•	Train models either locally or by using cloud resources, including GPU-accelerated model training.
•	Use automated machine learning, which accepts configuration parameters and training data. It automatically iterates through algorithms and hyperparameter settings to find the best model for running predictions.
•	Deploy web services to convert your trained models into RESTful services that can be consumed in any application.

**Comment by drguthals** [comment]
@KimberlyBolton1 
I'm not 100% sure what I'm going to do yet, but yes, I think these three I'm likley to for sure do:

> • Explore, prepare and manage the lifecycle of your datasets used in machine learning experiments.
> • Train models either locally or by using cloud resources, including GPU-accelerated model training.
> • Use automated machine learning, which accepts configuration parameters and training data. It automatically iterates through algorithms and hyperparameter settings to find the best model for running predictions.

But the other 2 I'm not positive yet, so let's leave those out. 

Thanks!

**Comment by KimberlyBolton1** [comment]
@sguthals
Edited abstract for your review:
Introducing the Azure ML SDK
The Azure Machine Learning (ML) SDK creates a way for developers to remain in the familiar environment of VS Code while using the power of Azure to run ML experiments and even deploy ML models. In this session, we’ll explore the lifecycle of datasets used in ML experiments, train models using local or cloud-based resources, and use automated ML.

**Comment by KimberlyBolton1** [comment]
@sguthals
Edited abstract for your review:
Introducing the Azure ML SDK
The Azure Machine Learning (ML) SDK creates a way for developers to remain in the familiar environment of VS Code while using the power of Azure to run ML experiments and even deploy ML models. In this session, we’ll explore the lifecycle of datasets used in ML experiments, train models using local or cloud-based resources, and use automated ML.

**Comment by drguthals** [comment]
Love it! 🚢  (the ship means "ship it", like, good to go ;) )

**Comment by KimberlyBolton1** [comment]
@mbaumb
Abstract final and moved to Ready for Sharepoint.

**Comment by drguthals** [comment]
@KimberlyBolton1 - Make sure you don't prematurely close issues 😉

**Comment by drguthals** [comment]
@mebaumb - Are you waiting for something else before we can confirm this in sharepoint?

**Comment by mebaumb** [comment]
@sguthals I didn't realize it had been through the marketing edits. I saw it in the done section and moved back over and was going to ask about this tomorrow. 

I will get it into sharepoint and confirmed tomorrow! Thank you!

**Comment by drguthals** [comment]
Ahhhhh good point - it was accidentally closed and then I re-opened but forgot to move it to the correct column! Good learning!

**Comment by mebaumb** [comment]
Published in SharePoint 6/23

**Comment by KimberlyBolton1** [comment]
Session recorded and uploaded to YouTube: https://youtu.be/BMbl_navbJw. Issue closed.
--
